pyiron_base/job/core.py

```diff
@@ -733,24 +734,42 @@ def _internal_copy_to(self, project=None, new_job_name=None, new_database_entry=
             )
         return new_job_core, file_project, hdf5_project, False
 
-    def copy_to(self, project, new_job_name=None, new_database_entry=True, copy_files=True):
+    def copy_to(self, project, new_job_name=None, input_only=False, new_database_entry=True, copy_files=True):
         """
         Copy the content of the job including the HDF5 file to a new location
 
         Args:
             project (JobCore/ProjectHDFio/Project): project to copy the job to
-            new_database_entry (bool): [True/False] to create a new database entry - default True
+            new_job_name (str): The new name to assign the duplicate job. Required if the project is `None` or the same
+                project as the copied job. (Default is None, try to keep the same name.)
+            input_only (bool): [True/False] Whether to copy only the input. (Default is False.)
+            new_database_entry (bool): [True/False] Whether to create a new database entry. If input_only is True then
+                new_database_entry is False. (Default is True.)
             copy_files (bool): [True/False] copy the files inside the working directory - default True
 
         Returns:
             JobCore: JobCore object pointing to the new location.
         """
-        new_job_core, _, _, _ = self._internal_copy_to(
+        # Update flags
+        if input_only and new_database_entry:
+            warnings.warn("input_only conflicts new_database_entry; setting new_database_entry=False")
+            new_database_entry = False
+
+        new_job_core, _, _, reloaded = self._internal_copy_to(
             project=project,
             new_job_name=new_job_name,
             new_database_entry=new_database_entry,
             copy_files=copy_files
         )
+        if reloaded:
+            return new_job_core
+
+        # Remove output if it should not be copied
+        if input_only:
+            for group in new_job_core.project_hdf5.list_groups():
+                if "output" in group:
+                    del new_job_core.project_hdf5[posixpath.join(new_job_core.project_hdf5.h5_path, group)]
+            new_job_core.status.initialized = True
```

Comment:
The job status of a job core object is just a string, the job status object is only introduced for the generic jobs.